src/amo/components/PromotedAddonsCard/index.js

```diff
@@ -0,0 +1,68 @@
+/* @flow */
+import makeClassName from 'classnames';
+import * as React from 'react';
+import { compose } from 'redux';
+
+import AddonsCard from 'amo/components/AddonsCard';
+import { LANDING_PAGE_PROMOTED_EXTENSION_COUNT } from 'amo/constants';
+import translate from 'core/i18n/translate';
+import type { AddonType } from 'core/types/addons';
+import type { I18nType } from 'core/types/i18n';
+
+import './styles.scss';
+
+type Props = {|
+  addonInstallSource?: string,
+  addons?: Array<AddonType> | null,
+  className?: string,
+  loading: boolean,
+|};
+
+export type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+|};
+
+export class PromotedAddonsCardBase extends React.Component<InternalProps> {
+  render() {
+    const { addonInstallSource, addons, className, i18n, loading } = this.props;
+
+    const header = (
+      <>
+        <span className="PromotedAddonsCard-headerText">
+          {i18n.gettext('Sponsored extensions')}
+        </span>
+        <a
+          className="PromotedAddonsCard-headerLink"
+          href="https://support.mozilla.org/kb/recommended-extensions-program"
+          rel="noopener noreferrer"
```

Comment:
This seems important to me, having a test case for it would be nice.

Reply:
I didn't add one because it is essentially all static, and we tend not to test things that are static. I guess I can add a test to check a part or some parts of this in case someone changes the code so that this is no longer passed to the `header` of `AddonsCard`.

Reply:
Yeah but this is a security feature. We shouldn't remove that accidentally.

Reply:
Oh! You specifically mean the `rel="noopener noreferrer"` part? I thought you were referring to the whole `header`.

Reply:
No, sorry only the `rel` attribute. I fully agree with you for the `header` being static.